src/portal/lib/src/replication/replication.component.ts

```diff
@@ -203,11 +203,11 @@ export class ReplicationComponent implements OnInit, OnDestroy {
 
     let params: RequestQueryParams = new RequestQueryParams();
     // Pagination
-    params.set("page", "" + pageNumber);
-    params.set("page_size", "" + this.pageSize);
+    params = params.set("page", "" + pageNumber);
```

Comment:
why need to resign params again?

Reply:
Because the RequestQueryParams object extend the new httpParams, the httpParams ‘s method will get a new object and will not change itself, so I resign params here.